Signal/src/util/OWSBackupAPI.swift

```diff
@@ -0,0 +1,245 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+import Foundation
+import SignalServiceKit
+import CloudKit
+
+@objc public class OWSBackupAPI: NSObject {
+    @objc
+    public class func recordIdForTest() -> String {
+        return "test-\(NSUUID().uuidString)"
+    }
+
+    @objc
+    public class func saveTestFileToCloud(fileUrl: URL,
+                                          success: @escaping (String) -> Swift.Void,
+                                          failure: @escaping (Error) -> Swift.Void) {
+        saveFileToCloud(fileUrl: fileUrl,
+                        recordName: NSUUID().uuidString,
+                        recordType: "test",
```

Comment:
Record name and types are part of the CloudKit API.  Although the record names necessarily leak out of this class - they are the identifier we use for our backup fragments - record types should be a private implementation detail of this class.  

Also, in the second PR I decided to use a single record type for all of our uploads (or "records" to use CloudKit parlance) since it facilitates clean up, and using distinct record type wasn't really buying us anything.